package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -381,3 +382,63 @@ def _topology_to_chemfiles(self, obj, n_atoms):
             topology.add_bond(bond.atoms[0].ix, bond.atoms[1].ix)
 
         return topology
+
+
+if HAS_CHEMFILES:
```

Comment:
why define the class only if chemfiles was found?

Reply:
Because it is inherited from `chemfiles.Trajectory`. If chemfiles is not present, `import MDAnalysis` will fail.
```python
  File "/home/scottzhuang/mdanalysis/package/MDAnalysis/coordinates/__init__.py", line 715, in <module>
    from . import chemfiles
  File "/home/scottzhuang/mdanalysis/package/MDAnalysis/coordinates/chemfiles.py", line 387, in <module>
    class ChemfilesPicklable(chemfiles.Trajectory):
NameError: name 'chemfiles' is not defined
```

Reply:
Sorry, I missed that point. Then the code looks good =)